tool/typedoc-theme.js

```diff
@@ -258,11 +163,28 @@ title: ${JSON.stringify(`${page.model.name} | JS API`)}
 // annotations as @-tags rather than needing to write out the HTML by hand.
 
 exports.load = app => {
-  app.renderer.addUnknownSymbolResolver('immutable', (name) =>
-      `https://immutable-js.com/docs/latest@main/${name}/`);
-  app.renderer.addUnknownSymbolResolver('source-map-js', (name) => {
-    if (name === 'RawSourceMap') {
-      return 'https://github.com/mozilla/source-map/blob/58819f09018d56ef84dc41ba9c93f554e0645169/source-map.d.ts#L15-L23';
+  app.converter.addUnknownSymbolResolver((ref, refl, part, symbolId) => {
```

Comment:
The CI error points to this line. IIUC that's because the other PR hasn't been merged, right? If so, everything LGTM

Reply:
Yeah, now that I've landed the other CL this one should pass CI.